src/Compilers/Core/CommandLine/BuildProtocol.cs

```diff
@@ -108,7 +109,6 @@ internal class BuildRequest
             for (int i = 0; i < args.Count; ++i)
             {
                 var arg = args[i];
-                Log($"argument[{i}] = {arg}");
```

Comment:
Why stop logging this?

Reply:
Logging these arguments ends up taking up 95% of the log file. It basically makes the file unreadable without massive filtering. The arguments are all present in the binlog so it doesn't really add anything here.